homework.md

```diff
@@ -92,13 +94,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+The "commit" command is used to save your changes to the local repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
```

Comment:
Hi Catalina well done, just wondering what happened to question 17. I don't see it

Reply:
Hi Tersia! Question 17 was there hiding. I just discovered expand all button.

Reply:
thanks for letting me know, I didn't know there was an expand button